apps/settings/elements/root.html

```diff
@@ -24,37 +24,37 @@ <h2 data-l10n-id="networkAndConnectivity">Network &amp; Connectivity</h2>
           </label>
         </li>
         <li data-category="connectivity">
-          <a id="menuItem-wifi" class="menu-item" data-icon="wifi-4" href="#wifi">
+          <a id="menuItem-wifi" class="menu-item" data-icon="wifi-4" href="#wifi" data-l10n-id="wifiSection" aria-describedby="wifi-desc">
```

Comment:
`aria-hidden="true"` seems make more sense to me as the icon actually presents the menu item described by the following span element. What do you think?